client/src/pages/messaging/index.js

```diff
@@ -47,6 +48,13 @@ const Chat = () => {
     storeUserSessionID(channelID, userId);
   }, [ channelID ]);
 
+  //The 'updateMessages' function makes the dependencies of useCallback Hook (at line 124) change on every render.
```

Comment:
Instead of changing `setMessages`, can we add another `useEffect` hook to update storage? 
```
useEffect(()=> {
   // store the messages
}, [messages])
```
In this way, changes will be minimal. Also, let's add a config variable here. We don't want to store messages by default.